### PR TITLE
Update to use the new release flow

### PR DESCRIPTION
**Pull request by BirmacherAkos**
- update to the new release flow
- remove gows dependecy management; remove old step.sh

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "colorstring",
+    "command",
+    "errorutil",
+    "fileutil",
+    "log",
+    "pathutil",
+    "retry",
+  ]
+  pruneopts = "UT"
+  revision = "6eb582e08179a025ed2deb86ea6dffbd13ba1ff6"
```
Comment (lpusok) [inline Gopkg.lock]: Is this the master branch's head?

File: bitrise.yml [hunk 2]
```diff
@@ -257,47 +256,3 @@ workflows:
             #!/bin/bash
             set -ex
             stepman audit --step-yml ./step.yml
-
```
Comment (lpusok) [inline bitrise.yml]: create-release workflow not needed

Comment (BirmacherAkos) [reply bitrise.yml]: Yes, it is:
https://github.com/bitrise-io/go-utils/commit/6eb582e08179a025ed2deb86ea6dffbd13ba1ff6

Go packages don't have releases this is why it's locked on a commit hash.

File: Gopkg.lock (hunk 1 above)
Comment (BirmacherAkos) [reply Gopkg.lock]: Yes, it is:
https://github.com/bitrise-io/go-utils/commit/6eb582e08179a025ed2deb86ea6dffbd13ba1ff6

Go packages don't have releases this is why it's locked on a commit hash.

File: bitrise.yml (hunk 2 above)
Comment (trapacska) [reply bitrise.yml]: Switch godeps update with dep update workflow

**Pull request merged**